packages/framework/fluid-framework/api-report/fluid-framework.api.md

```diff
@@ -250,6 +692,226 @@ export interface IDisposable {
     [disposeSymbol](): void;
 }
 
+// @public
+export interface IDisposable_2 {
```

Comment:
This seems unfortunate that IDisposable and IDisposable_2 are different?  Is there something we can do to disambiguate these more nicely, merge them, etc.?

Reply:
We have existing internal work items about that, see https://dev.azure.com/fluidframework/internal/_workitems/edit/6591/ which I have also just updated.